README.rst

```diff
@@ -41,13 +41,32 @@
 MAPIE - Model Agnostic Prediction Interval Estimator
 ====================================================
 
-**MAPIE** allows you to easily estimate prediction intervals (or prediction sets)
-using your favourite scikit-learn-compatible model for
-single-output regression or multi-class classification settings.
+Quantifying the uncertainties and controlling the risks of ML model predictions is of crucial importance
+for developing and deploying reliable artificial intelligence (AI) systems. Uncertainty quantification (UQ)
+involves all the stakeholders who develop and use AI models.
 
-Prediction intervals output by **MAPIE** encompass both aleatoric and epistemic
-uncertainties and are backed by strong theoretical guarantees thanks to conformal
-prediction methods [1-7].
+**MAPIE** is an open-source Python library hosted on scikit-learn-contrib project that allows you to:
```

Comment:
Maybe we could add links to the places in the documentations where we "wrap your favorite scikit-learn-compatible model".

Reply:
I will make a proposition.

Reply:
I try to add a hyperlink in the file.